requirements.txt

```diff
@@ -2,4 +2,4 @@ ntplib
 simplejson
 tornado
 pyyaml
-docker-py
+docker-py==1.10.6
```

Comment:
We should pin this to the version that's used in the agent if we're going to pin it